webdriver-spec.html

```diff
@@ -699,13 +699,18 @@
 <section>
 <h2>Interface</h2>
 
-<pre class=idl>partial interface Navigator {
-	[Unforgeable] readonly attribute boolean webdriver;
+<p>If the <dfn>webdriver-active flag</dfn> is true,
```

Comment:
As I understand from https://w3c.github.io/webdriver/webdriver-spec.html#new-session there is a hypothetical possibility to attach to a browser instance at runtime. 
> WebDriver implementations typically start a completely new browser instance, but there is no requirement in this specification (or for WebDriver only to be used to automate only web browsers). Implementations might choose to use an existing browser instance, eg. by selecting the window that currently has focus.

Implementations that I looked at (WebKit, Chrome) require new instance.